ui-common/src/main/java/io/syndesis/qe/utils/ByUtils.java

```diff
@@ -34,7 +34,14 @@ public static By rowid(String rowId) {
     }
 
     public static By partialLinkText(String partialText) {
-        return new ByPartialText(partialText);
+        return new By() {
+            @Override
+            public List<WebElement> findElements(SearchContext searchContext) {
+                return searchContext.findElements(By.cssSelector("a,button")).stream()
```

Comment:
button can be also <input type="button"> and <input type="submit"> if we want to have all link-ish elements covered